crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -242,7 +242,7 @@ impl FormatNodeRule<Parameters> for FormatParameters {
             + usize::from(kwarg.is_some());
 
         if self.parentheses == ParametersParentheses::Never {
-            write!(f, [group(&format_inner)])
+            write!(f, [group(&format_inner), dangling_comments(dangling)])
```

Comment:
If the lambda does have parameters, and those parameters have dangling comments, they're now handled by the lambda directly.

Reply:
If the lambda doesn't have parameters, and itself has dangling comments, those are formatted automatically by the lambda as "standard" dangling comments.

Reply:
Formatting dangling comments of another node can be problematic if we e.g. want to support suppression of any node, because it is impossible to *undo* the formatting that was done by the parent. 

Is there another way to get the desired formatting that leaves the dangling comment formatting inside of `Parameters`?

Reply:
Hmm, but the point is that the parent only handles dangling comments if the parameters don't exist at all (are `None`).